src/main/java/roomescape/member/dto/MemberResponse.java

```diff
@@ -1,8 +1,11 @@
 package roomescape.member.dto;
 
+import io.swagger.v3.oas.annotations.media.Schema;
 import roomescape.member.model.Member;
 
-public record MemberResponse(Long id, String name, String email) {
+public record MemberResponse(Long id, String name,
+                             @Schema(example = "user@mail.com")
+                             String email) {
```

Comment:
```suggestion
public record MemberResponse(
    Long id, 
    String name,
    @Schema(example = "user@mail.com") String email
) {
```
한줄에 하나만 오도록 해주세요!
어던 파라미터가 있는지 파악하기 어렵습니다!